integration/test-project/.graphqlrc.yml

```
@@ -1,20 +1,8 @@
 schema: ./server/src/schema/**/*.graphql
-documents: ./client/src/graphql/**/*.graphql
 extensions:
   codegen:
     generates:
-      ./client/src/generated-types.tsx:
-        config:
-          withComponent: false
-          withHOC: false
-          withHooks: true
-          skipDocumentsValidation: true
-        plugins:
-          - add: '/* tslint:disable */'
-          - typescript
-          - typescript-operations
-          - typescript-react-apollo
-      ./server/src/generated-types.ts:
+      ./src/generated-types.ts:
         config:
           contextType: '@graphback/runtime#GraphbackRuntimeContext'
```

`@graphback/runtime#GraphbackRuntimeContext` package interface does not exist any more. `GraphbackContext` now sits in `@graphback/core`, so maybe:

```suggestion
          contextType: '@graphback/core#GraphbackContext'
```